x-pack/plugin/security/qa/operator-privileges-tests/src/javaRestTest/java/org/elasticsearch/xpack/security/operator/Constants.java

```diff
@@ -69,6 +69,7 @@ public class Constants {
         "cluster:admin/scripts/painless/execute",
         "cluster:admin/synonyms/delete",
         "cluster:admin/synonyms/get",
+        "cluster:admin/synonyms/list",
```

Comment:
Should we modify this to `cluster:admin/synonymsset/get` ?

Reply:
Makes sense!